sycl/test-e2e/Matrix/joint_matrix_int8_rowmajorA_rowmajorB.cpp

```diff
@@ -1,19 +1,16 @@
-//==-------- joint_matrix_bf16_vnni.cpp  - DPC++ joint_matrix---------------==//
+//==----- joint_matrix_int8_rowmajorA_rowmajorB.cpp  - DPC++ joint_matrix---==//
 //
 // Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
 // See https://llvm.org/LICENSE.txt for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
-// REQUIRES: matrix
+// REQUIRES: aspect-ext_intel_matrix
 
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 // RUN: %if gpu %{ env IGC_JointMatrixLoadStoreOpt=0 %{run} %t.out %}
 // RUN: %if gpu %{ env IGC_JointMatrixLoadStoreOpt=1 %{run} %t.out %}
```

Comment:
I see these 2 lines will fail in pre-check-in when older IGC is used. I suggest to modify it as follows:
```suggestion

// Run these 2 tests on PVC only for now. Check can be updated to "gpu", when newer IGC is used in intel/llvm pre-checkin testing on Intel Arc
// RUN: %if gpu-intel-pvc %{ env IGC_JointMatrixLoadStoreOpt=0 %{run} %t.out %}
// RUN: %if gpu-intel-pvc %{ env IGC_JointMatrixLoadStoreOpt=1 %{run} %t.out %}
```